### DISCUSSION TITLE
I send an email from the support of the npm website, but it always returns a 403 error .

### DISCUSSION THREAD

**Discussion by duan602728596** [npm/feedback#662] [Website] [answered]
I send an email from the support of the npm website, but it always returns a 403 error .
I have tried various network environments and browsers, but cannot send mail successfully. 
<img width="1320" alt="截屏2022-04-05 上午1 19 09" src="https://user-images.githubusercontent.com/17332657/161597427-d29376cf-8ede-4946-a2bd-e95ebccf3f90.png">

**Comment by MylesBorins** [comment, marked as answer]
@duan602728596 I just tried it an was able to successfully submit, can you try again and lmk if you are still having this issue

**Reply by duan602728596** [reply]
> @duan602728596 I just tried it an was able to successfully submit, can you try again and lmk if you are still having this issue

I tried to resend, still the problem.

**Reply by duan602728596** [reply]
> @duan602728596 I just tried it an was able to successfully submit, can you try again and lmk if you are still having this issue

I tried changing the proxy channel and it sent successfully. Sorry, due to regional reasons.

**Reply by MylesBorins** [reply]
Found your support ticket and fixed the problem you were running into 👍🏼
